testData/solutions/win32.json5

```diff
@@ -2617,6 +2660,44 @@
                             "default": 0
                         }
                     },
+                    "OutputModes.ACCESS_KEY": {
+                        "schema": {
+                            "title": "Access Key",
+                            "description": "Which items to speak with the access key???",
+                            "type": "string",
+                            "default": "1|1|1|Access Key",
+                            "enum": [
+                                "0|0|0|Access Key",
```

Comment:
Is this literally what's stored in the settings file?

Reply:
Yup, it's the actual string, of three matching numbers and the label separated by pipes. I just copied them straight from the .ini file format after saving the dialog with different options. It's admittedly pretty wonky.

Reply:
As long as we're clear with Gregg that we can only really handle them as enums for now, it doesn't matter how funny they get.  They can use the table flipping ascii image as "false" for all we care.

Reply:
@the-t-in-rtf I think it's ok. It is an enum, so this odd string isn't showing up in any of the user facing tools.

Reply:
Yeah, if all of these are the literal values that are expected to be written to the INI file, then I'm fine.  Just wanted to confirm as with the others.